### DISCUSSION TITLE
Emails sent from exchange like o365 are redelivered if postal can't deliver it first

### DISCUSSION THREAD

**Discussion by chrism417** [postalserver/postal#2112] [Help with using Postal]
If Rabbit goes down, emails show as "Pending" in Postal, however exchange retries every minute and every retry is a new email.

Is there any way for postal to ACK the message since technically it can be redelivered (at least, the button is there, so it's stored)?

**Comment by willpower232** [comment]
I'm confused why exchange is retrying a message Postal has already accepted? there must be some miscommunication somewhere. Are you able to see in the SMTP logs how they interacted?
